galley/pkg/metadata/kube/types.go

```diff
@@ -468,16 +468,16 @@ func init() {
 		Converter: converter.Get("identity"),
 	})
 
-	b.Add(kube.ResourceSpec{
-		Kind:      "handler",
-		ListKind:  "handlerList",
-		Singular:  "handler",
-		Plural:    "handlers",
-		Version:   "v1alpha2",
-		Group:     "config.istio.io",
-		Target:    metadata.Types.Get("type.googleapis.com/istio.policy.v1beta1.Handler"),
-		Converter: converter.Get("identity"),
-	})
+	// b.Add(kube.ResourceSpec{
```

Comment:
@douglas-reid @ayj please file an issue about this. Galley/Mixer need to handle custom struct form in protos.

Reply:
https://github.com/istio/istio/issues/8486